pkg/util/conditions/condition_types.go

```diff
@@ -38,3 +42,8 @@ const (
 	FetchingSecretRefSuccessReason ConditionReason = "FetchingSecretRefSuccess"
 	FetchingSecretRefFailedReason  ConditionReason = "FetchingSecretRefFailed"
 )
+
+const (
+	GarmServerNotReconciledYet ConditionMessage = "GARM server not reconciled yet"
+	DeletingEnterprise         ConditionMessage = "Deleting enterprise"
```

Comment:
If we want to have a `DeletingEnterprise` condition message here, we should also include one for Orgs, Repos and Pools as well